### ISSUE TITLE
Menu entry "assign tags" gone

### ISSUE THREAD

**Issue by MelwinQ** [theCrag/website#2297]
At the sector, cliff and crag level, the Action menu does not contain the entry "Assign tags" anymore. I know I can reach this via "Bulk edit", but find this a bit cumbersome.

**Comment by scd** [comment]
We are removing forms where functionality exists in the bulk edit for code maintenance reasons. 

I would like to understand more specifics of what you mean by cumbersome in the Bulk edit. If it is something that is fixable I would rather fix that.

I am also thinking that if we are going to have a second way of updating tags then maybe it should be directly on the area page itself via ajax and javascript.

If we have to bring the assign tags form back then I have to schedule this work because it now requires some translations upgrades.

**Comment by MelwinQ** [comment]
Generally, I find the tagging option quite useful for providing structured additional information. So far, I did not use it as much as I should, as assigning tags was already a bit hidden (though it still could be reached by going into one dropdown). 
Currently, I just discovered the option of changing tags by mere chance; I wouldn't have have sought it under "bulk edit". Moreover, it now requires to go trough this "prior-page" (for specifying the displayed fields), making the entire process more indirect.
Baseline, I think a more direct ajax/js solution would promote the use of tags. Thus, it would also address #1713.
